README.md

```diff
@@ -33,36 +33,50 @@ module "example" {
-| role_tags | Tags to apply to the IAM role created. | `map(string)` | `{}` | no |
-| terraform_account_name | The name of the account containing the S3 bucket where Terraform state is stored. | `string` | `Terraform` | no |
-| terraform_state_bucket_name | The name of the S3 bucket where Terraform state is stored (e.g. example-terraform-state-bucket). | `string` | n/a | yes |
-| terraform_state_path | The path to the Terraform state key(s) in the S3 bucket that the role will be allowed to read (e.g. example-terraform-project/*). | `string` | n/a | yes |
-| terraform_workspace | The name of the workspace containing the Terraform state that the role will be allowed to read.  Defaults to all workspaces ('*'). | `string` | `*` | no |
+| account\_ids | AWS account IDs that are allowed to assume the role that allows read-only access to the specified Terraform state. | `list(string)` | `[]` | no |
+| assume\_role\_policy\_description | The description to associate with the IAM policy that allows assumption of the role that allows read-only access to the specified Terraform state.  Note that the first "%s" in this value will get replaced with the role\_name variable and the second "%s" will get replaced with the terraform\_account\_name variable. | `string` | `"Allow assumption of the %s role in the %s account."` | no |
+| assume\_role\_policy\_name | The name to assign the IAM policy that allows assumption of the role that allows read-only access to the specified Terraform state.  Note that the "%s" in this value will get replaced with the role\_name variable. | `string` | `"Assume%s"` | no |
+| aws\_region | The AWS region where the non-global resources are to be provisioned (e.g. "us-east-1"). | `string` | `"us-east-1"` | no |
+| iam\_usernames | The list of IAM usernames allowed to assume the role that allows read-only access to the specified Terraform state.  If not provided, defaults to allowing any user in the specified account(s).  Note that including "root" in this list will override any other usernames in the list. | `list(string)` | `["root"]` | no |
+| role\_description | The description to associate with the IAM role (as well as the corresponding policy) that allows read-only access to the specified state in the specified S3 bucket where Terraform state is stored.  Note that the first "%s" in this value will get replaced with the terraform\_state\_path variable, the second "%s" will get replaced with the terraform\_workspace variable, and the third "%s" will get replaced with the terraform\_state\_bucket\_name variable. | `string` | `"Allows read-only access to the Terraform state at '%s' for the '%s' workspace(s) in the %s S3 bucket."` | no |
+| role\_name | The name to assign the IAM role (as well as the corresponding policy) that allows read-only access to the specified state in the S3 bucket where Terraform state is stored. | `string` | n/a | yes |
+| role\_tags | Tags to apply to the IAM role created. | `map(string)` | `{}` | no |
```

Comment:
`role_tags` situation is in this repository as well.

Reply:
Yep, that is the PR I will create after this one is merged.

Reply:
See PR #5.